earth_enterprise/src/fusion/autoingest/AssetStorage.idl

```diff
@@ -88,11 +88,11 @@ class AssetVersionStorage {
   AssetDefs::State state;
   double       progress;
   bool         locked;
-  std::vector<std::string> inputs;
-  std::vector<std::string> children;
-  std::vector<std::string> parents;
-  std::vector<std::string> listeners;
-  std::vector<std::string> outfiles;
+  MTVector<std::string> inputs;
```

Comment:
IIRC `inputs` was the attribute I first hit problems with during parallel builds.  I made all 5 use the new type at the same time just to be safe.